adversarial_text/adversarial_losses.py

```diff
@@ -24,7 +24,7 @@
 FLAGS = flags.FLAGS
 
 # Adversarial and virtual adversarial training parameters.
-flags.DEFINE_float('perturb_norm_length', 0.1,
+flags.DEFINE_float('perturb_norm_length', 5.0,
```

Comment:
Could you also add a comment above the embedding keep prob FLAG in graphs.py about what it should be to reproduce the results?